source/codegen/metadata/nifgen/functions.py

```diff
@@ -726,26 +726,6 @@
         ],
         'returns':'ViStatus'
     },
-    'ConfigureGain':{
```

Comment:
CHANGES.md documents any changes we are making w.r.t. base metadata. We should document these functions as removed because they were obsolete.

Reply:
updated Changes.md